.browserslistrc

```diff
@@ -36,27 +36,6 @@ opera 76
 safari 14
 samsung 13.0
 
-# Same as `stable` but with IE 11
-
-[legacy]
-ie 11
-and_chr 91
```

Comment:
On a similar note, should we keep updating the versions for every major release as we did (https://github.com/mui/mui-x/pull/2384) for v5 (and skipped for v6...)?
cc @DanailH @oliviertassinari

Reply:
Yes, I think we should agree with Material UI on what would be the broowserlist for  Material UI v6 and start to using it in MUI X https://github.com/mui/material-ui/issues/40958#issuecomment-1951351873. This way, we guarantee eventual consistency.